### DISCUSSION TITLE
removed mandatory rule in BT-746-Organization and it's consequences

### DISCUSSION THREAD

**Discussion by gabrieledall** [OP-TED/eForms-SDK#778] [Q&A] [answered]
Hello OP,

while scanning the changes in SDK-1.10.0, we also came through the following changed rule "Removed the mandatory rule on "Listed on a Regulated Market" (BT-746-Organization)".

In Gemany this rule is expected to remain mandatory and therfore we will have to implement it now in the german rule set by our own. This is not the motivation of my question. The question to us is if there are other implicit assumptions that made you remove this rule.

Especially those that might imply the existence of Contracts in CANs. By now, we assume, that contracts are required to identify the "real" winners of a procurement decision. Because contracts refer to tenders and tenders held the contracting parties. Without contracts noc winners. 

Will this assuption still hold or are winners sooner or later be identifiable only from the LotTenders-Block assuming that only the winning LotTenders will be reported with a CAN? If this would be true in the long run, we need to change the rule in BT-746 to make sure BT-746 is only reuired be be filled in for the winning party organizations.

Thank you for your answer.

Kind regards
Gabriele

**Comment by YvesJo** [comment]
Hi,
The rule on BT-746-Organization was defined to ensure information consistency between different properties of an organization. This was however disturbing for multiple stakeholders and has been removed. You're right when saying that "without contract, there is no winner".
It is, and will still be, possible to report information about non-winning tenders and the associated objects like Tendering Parties and the organizations that compose them. Also keep in mind that the organization properties grouped under the Business Group "Winner" (BG-4) does not only apply to organizations that submitted a winning tender but to "winner(s), tenderer(s) or subcontractor(s)." as stated in the descriptions of the Business Group (BG-4) and its included Business Terms (BT-165, BT-706, BT-746).
I hope this helps answer your question.
KR

**Comment by gabrieledall** [comment, marked as answer]
thank you. Your answer helps a lot!
